amp/TEMPLATE/ampTemplate/gisModule/dev/app/js/amp/map/views/structures-view.js

```diff
@@ -203,10 +203,14 @@ module.exports = Backbone.View
         }
       }
     }
-    
-    var icon = this.structureMenuModel.iconMappings[sectorCode] || this.structureMenuModel.iconMappings[this.structureMenuModel.DEFAULT_ICON_CODE];    
-    var pointIcon = L.icon({
-      iconUrl: 'img/map-icons/' + icon,
+
+    var sectorSvgCode = this.structureMenuModel.DEFAULT_ICON_CODE;
```

Comment:
Lets try to create a function to decide the sector code so we avoid having code duplicated